src/scripts/validation.js

```diff
@@ -52,6 +52,11 @@ const toggleButtonState = (inputList, buttonElement, config) => {
   }
 };
 
+export const disableSubmitButton = (buttonElement, config) => {
+  buttonElement.classList.add(config.inactiveButtonClass);
+  buttonElement.disabled = true;
+};
+
```

Comment:
**Could be improved**


You can make a `reset` listener for **disabling** the submit button:


```js

  // here you disable the button when you start the project
  toggleButtonState(inputList, buttonElement, config);

  // here you add the `reset` handler
  formElement.addEventListener("reset", () => {
    disableSubmitButton(buttonElement, config) 
  });

```

In this `reset` listener we are waiting for a special `reset` event which is fired when you `reset` any form